content/articles/cryptanalysis-of-data-after-a-ransomware-attack/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: engineering-education
+status: publish
+published:
+url: /cryptanalysis-of-data-after-a-ransomware-attack/
+title: Cryptanalysis of Data after A Ransomware Attack
+description: This article seeks to explore the concept of cryptanalysis in light of data encrypted by a malicious actor seeking ransom to decrypt the data. The process of cryptanalysis will be clearly outlined, highlighting the various ethical ways to do the same putting into consideration the possible cryptographic algorithm that may have been employed in encrypting the data.
+
+The possible decryption algorithms will also be identified to be able to help the reader know their possible go-to option when they fall prey to a ransomware attack.
+author: ruth-mare
+date: 2021-08-06T00:00:00-18:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/cryptanalysis-of-data-after-a-ransomware-attack/hero.jpg
+    alt: Cryptanalysis cover image 
+---
+Cryptanalysis is a technique that has been majorly applied to helping find weaknesses in cryptographic algorithms and improve on them to make encryption robust to threats. Cryptanalysis can, however, be used to decipher data encypted by malicous actors to save on the cost of paying the ransom and recover confidential data.
+<!--more-->
+The concepts described in this article will help understand this process.
+
+### Overview
+This article will cover:
+- [Prerequisites](#prerequisites)
+- [What is cryptanalysis?](#what-is-cryptanalysis?)
```

Comment:
Please remove the question mark in your link description as it is breaking the link.